cmds/update.js

```diff
@@ -0,0 +1,321 @@
+/* eslint-disable no-param-reassign */
+const inquirer = require('inquirer');
+const { format } = require('date-fns');
+const connection = require('../lib/ezunpaywall');
+const { getConfig } = require('../lib/config');
+const logger = require('../lib/logger');
+
+const {
+  getState,
+  verbose,
+  getSnapshots,
+  getReport,
+  getReports,
+  force,
+} = require('../bin/update');
+
+/**
+ * Starts an unpaywall data update process
+ *
+ * @param {String} option.file --file <file> | snapshot's file installed on ezunpaywall
+ * and insert updates from unpaywall
+ * @param {String} option.offset --offset <offset> | line where processing will start
+ * @param {String} option.limit --limit <limit> | line where processing will end
+ * @param {Boolean} option.list -L --list | list of snapshots installed on ezunpaywall
+ * @param {Boolean} option.index -I --index | name of the index to which the data is inserted
+ */
+const updateJobFile = async (option) => {
+  const ezunpaywall = await connection();
+  const config = await getConfig();
+
+  if (option.file) {
+    const pattern = /^[a-zA-Z0-9_.-]+(.gz)$/;
+    if (!pattern.test(option.file)) {
```

Comment:
You can make it simpler. Also don't forget that `.` is a special character in regular expressions.

```suggestion
    if (!/\.gz$/i.test(option.file)) {
```